pkg/apiserver/registry/intelligence/throughputanomalydetector/rest.go

```diff
@@ -34,6 +34,9 @@ import (
 const (
 	defaultNameSpace     = "flow-visibility"
 	tadQuery         int = iota
+	aggtadpodQuery
+	aggtadpod2podQuery
+	aggtadpod2svcQuery
```

Comment:
Keep the camel-case naming to make them `aggTadPodQuery`, `aggTadPod2PodQuery`, `aggTadPod2SvcQuery`